site/content/entry/citra-progress-report-2018-q2.md

```diff
@@ -0,0 +1,141 @@
++++
+date = "2018-07-27T12:21:53-05:00"
+title = "Citra Progress Report - 2018 Q2"
+tags = [ "progress-report" ]
+author = "anodium"
+forum = 0
++++
+
+Hello everyone to the summer (or winter, for those on the southern half of this
+Earth) edition of Citra's progess reports. Although not very many changes have
```

Comment:
I think "very many" is grammatically incorrect. Please consider replacing with "too many".

Reply:
It's correct, though not very smooth